Dist files missing from installed package

I'm getting the following error when trying to run this:

```
➜ spectral-language-server
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '/home/bsmith/.asdf/installs/nodejs/16.14.0/.npm/lib/node_modules/spectral-language-server/dist/spectral-language-server/server.js'
Require stack:
- /home/bsmith/.asdf/installs/nodejs/16.14.0/.npm/lib/node_modules/spectral-language-server/bin/spectral-language-server
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/bsmith/.asdf/installs/nodejs/16.14.0/.npm/lib/node_modules/spectral-language-server/bin/spectral-language-server:3:1)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/bsmith/.asdf/installs/nodejs/16.14.0/.npm/lib/node_modules/spectral-language-server/bin/spectral-language-server'
  ]
}
```

```
➜ l /home/bsmith/.asdf/installs/nodejs/16.14.0/.npm/lib/node_modules/spectral-language-server
total 20K
drwxr-xr-x   2 4.0K Feb 14 21:36 bin/
drwxr-xr-x 153 4.0K Feb 14 21:37 node_modules/
-rw-r--r--   1 1.1K Feb 14 21:36 LICENSE
-rw-r--r--   1  896 Feb 14 21:36 package.json
-rw-r--r--   1  624 Feb 14 21:36 README.md
```

The only explanation I've found so far is the `dist` directory being in the .gitignore file. According to the npm docs:
> If there is a .gitignore or .npmignore file, then ignored files in that and all child directories will be excluded from the package. If both files exist, then the .gitignore is ignored, and only the .npmignore is used.

Other sources say that `files` should override it, though.

--- a/package.json
+++ b/package.json
@@ -17,7 +17,8 @@
   "scripts": {
     "clean": "rimraf tmp/*",
     "release": "np --no-yarn",
-    "test": "echo 1"
+    "test": "echo 1",
+    "postinstall": "./script/vscode-spectral-build.sh"
   },
   "dependencies": {
     "@stoplight/spectral": "^5.8.0",

--- a/script/vscode-spectral-build.sh
+++ b/script/vscode-spectral-build.sh
@@ -18,7 +18,7 @@ git pull --rebase
 
 # npm install
 cd $DIR/../tmp/vscode-spectral
-yarn add gulp gulp-cli
+yarn config set ignore-engines true
 yarn
 yarn compile
 

